<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="71219" started-at="2019-07-20T13:31:06Z" finished-at="2019-07-20T13:32:17Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="71219" started-at="2019-07-20T13:31:06Z" finished-at="2019-07-20T13:32:17Z">
      <class name="contactpage.CheckErrorWithoutEnteringDetails">
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:contactpage.CheckErrorWithoutEnteringDetails@17d99928]" name="beforeMethod" is-config="true" duration-ms="65886" started-at="2019-07-20T19:01:06Z" finished-at="2019-07-20T19:02:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="testErrorWithoutFillingDetails()[pri:0, instance:contactpage.CheckErrorWithoutEnteringDetails@17d99928]" name="testErrorWithoutFillingDetails" duration-ms="5291" started-at="2019-07-20T19:02:12Z" finished-at="2019-07-20T19:02:17Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at global.Scroller.jsExecutor(Scroller.java:11)
	at contactpage.CheckErrorWithoutEnteringDetails.testErrorWithoutFillingDetails(CheckErrorWithoutEnteringDetails.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testErrorWithoutFillingDetails -->
      </class> <!-- contactpage.CheckErrorWithoutEnteringDetails -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
